docs/web-sdk/passing-a-custom-sourceNode.md

```diff
@@ -0,0 +1,106 @@
+---
```

Comment:
This file should be deleted. It contains Web SDK details which we are not pushing right now (as mentioned earlier).